waterlog/src/actions/Types.js

```diff
@@ -1,17 +1,20 @@
-export const FETCH_SENSORS = "FETCH_SENSORS";
-export const FETCH_SEGMENTS = "FETCH_SEGMENTS";
-export const FETCH_SEGMENTS_EVENTS = "FETCH_SEGMENTS_EVENTS";
-export const FETCH_SEGMENTS_LEAKS_BEGIN = "FETCH_SEGMENTS_LEAKS_BEGIN";
-export const FETCH_SEGMENTS_LEAKS_SUCCESS = "FETCH_SEGMENTS_LEAKS_SUCCESS";
-export const FETCH_SEGMENTS_LEAKS_FAILURE = "FETCH_SEGMENTS_LEAKS_FAILURE";
-export const FETCH_SEGMENTS_LEAK = "FETCH_SEGMENTS_LEAK";
-export const FETCH_COSTS = "FETCH_COSTS";
-export const FETCH_LEAK_LITRES_BEGIN = "FETCH_LEAK_LITRES_BEGIN";
-export const FETCH_LEAK_LITRES_SUCCESS = "FETCH_SEGMENTS_LEAKS_SUCCESS";
-export const FETCH_LEAK_LITRES_FAILURE = "FETCH_LEAK_LITRES_FAILURE";
-export const FETCH_SEGMENTS_RESOLVED_BEGIN = "FETCH_SEGMENTS_RESOLVED_BEGIN";
-export const FETCH_SEGMENTS_RESOLVED_SUCCESS = "FETCH_SEGMENTS_RESOLVED_SUCCESS";
-export const FETCH_SEGMENTS_RESOLVED_FAILURE = "FETCH_SEGMENTS_RESOLVED_FAILURE";
+export const FETCH_SENSORS = 'FETCH_SENSORS';
```

Comment:
in addition to @RashadAkoodieRabbit's comment, the consistency shouldn't just be in one file, it should be throughout the project. on the top file, you chose " " instead of ' ' but here you're going with ' ' instead of " "

Reply:
I fixed this.